### DISCUSSION TITLE
Point protoc_binary to a local copy

### DISCUSSION THREAD

**Discussion by PolarBear292208** [thought-machine/please#2277] [Q&A] [answered]
Is it possible to point protoc_binary to a local copy of protoc?

At the moment Google doesn't have Apple M1 binaries hosted on Github ([tracking issue](https://github.com/protocolbuffers/protobuf/issues/9397)) so I need to build a local copy. Is it possible to point the protoc_binary rule to it so I can compile my proto files in the meantime?

**Comment by tiagovtristao** [comment, marked as answer]
You can skip `protoc_binary` altogether and provide your protoc via the `ProtocTool` option in `.plzconfig`: https://please.build/config.html#:~:text=all%20of%20them.-,ProtocTool,-The%20binary%20invoked
